cmd/libs/go2idl/openapi-gen/.import-restrictions

```diff
@@ -0,0 +1,15 @@
+{
+  "Rules": [
+    {
+      "SelectorRegexp": "k8s[.]io",
+      "AllowedPrefixes": [
+        "k8s.io/kubernetes/cmd/libs/go2idl",
+        "k8s.io/apimachinery/pkg/openapi",
+        "k8s.io/gengo",
+        "k8s.io/kubernetes/third_party",
+        "k8s.io/apimachinery/third_party",
+        "k8s.io/apimachinery/pkg/util/sets"
+      ]
+    }
+  ]
+}
```

Comment:
oh, its missing a new line at the end.  Let's fix that - even github knows that's just wrong  :-)